core/src/main/java/tech/tablesaw/table/Relation.java

```diff
@@ -529,4 +532,54 @@ public String getString(int r, int columnIndex) {
   public boolean containsColumn(Column<?> column) {
     return columns().contains(column);
   }
+
+  /**
+   * Maps the supplied functions across all rows in the table appending the results to a new DoubleColumn.
+   *
+   * If you do not want the supplied function applied to rows with missing values you can supply a
+   * list of columns to check for missing values. For each row, it will check for a missing value in any of the
+   * listed columns. If there is a missing value a missing value will be appended to the results, otherwise
+   * the result of the function will be appended.
+   *
+   *
+   * @param name The name of the new column.
+   * @param function to map a table row to a double.
+   * @param skipIfMissingColNames the column names to check for missing values. If any of the
+   * columns contain a missing value (for a particular row) a missing value will be appended.
+   * @return DoubleColumn with the results appended.
+   */
+  public DoubleColumn mapToDoubleColumn(String name, Function<? super Row, ? extends Double> function, String... skipIfMissingColNames) {
+    return (DoubleColumn) mapToTypeInternal(DoubleColumn.create(name, rowCount()), function, skipIfMissingColNames);
+  }
+
+  private <T> Column<T> mapToTypeInternal(Column<T> destination,
+    Function<? super Row, ? extends T> function, String[] skipIfMissingColNames) {
+    List<Column<?>> checkMissingCols = Arrays.stream(skipIfMissingColNames).map(this::column).collect(Collectors.toList());
+    Iterator<Row> rowIterator = iterator();
+    int i = 0;
+    while(rowIterator.hasNext()) {
```

Comment:
did you rebase against `master`? I would think that the Google code formatter would add a space after `while`